src/commands/docker_buildx_image.yml

```diff
@@ -0,0 +1,203 @@
+---
+
+description: Build, tag and push Docker image to remote registry [AWS ECR].
```

Comment:
🐼  Maybe add "for cross platform" in the description to differentiate command from the standard `buil` command?